@package laravel/framework

--- src/Illuminate/Foundation/Testing/TestCase.php
+++ src/Illuminate/Foundation/Testing/TestCase.php
@@ -15,7 +15,7 @@
 use Mockery;
 use Mockery\Exception\InvalidCountException;
 use PHPUnit\Framework\TestCase as BaseTestCase;
-use PHPUnit\Util\Annotation\Registry;
+use PHPUnit\Metadata\Annotation\Parser\Registry;
 use Throwable;

 abstract class TestCase extends BaseTestCase
@@ -294,7 +294,7 @@ protected function callBeforeApplicationDestroyedCallbacks()
      * @param  \Throwable  $exception
      * @return void
      */
-    protected function onNotSuccessfulTest(Throwable $exception): void
+    protected function onNotSuccessfulTest(Throwable $exception): never
     {
         parent::onNotSuccessfulTest(
             is_null(static::$latestResponse)
